spec/services/container_deployment_service_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'recursive-open-struct'
+
+describe ContainerDeploymentService do
+  before do
+    %w(amazon openstack google azure redhat vmware).each do |p|
+      network = FactoryGirl.create(:network, :ipaddress => "127.0.0.1")
+      hardware = FactoryGirl.create(:hardware)
+      hardware.networks << network
+      vm = FactoryGirl.create("vm_#{p}".to_sym, :hardware => hardware)
+      if (p.include? "amazon") || (p.include? "redhat")
```

Comment:
we ask if `p` is one of this two phrases.
it's the same as asking if `p` is one `or` `p` is the other.

for example
```
irb(main):026:0> ['hello','world'].include?('somthing')
=> false
irb(main):027:0> ['hello','world'].include?('world')
=> true
irb(main):028:0> ['hello','world'].include?('hello')
=> true
```